recipes/cuda-cudart/post-link.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+[[ -d "$PREFIX/lib/pkgconfig" ]] && sed -E -i "s|cudaroot=.+|cudaroot=$PREFIX|g" $PREFIX/lib/pkgconfig/cuda*.pc
```

Comment:
Could we do this in `build.sh`? Conda-build will detect `$PREFIX` and handle correct replacement for us during install. That would simplify things here a bit

Reply:
Done, thanks!